### PR TITLE
Last contact

### PR DESCRIPTION
**Pull request by ianlovell**

### CODE REVIEW

File: app/actions/index.js [hunk 1]
```diff
@@ -7,40 +9,35 @@ export const UPDATE_SESSIONS_START = 'UPDATE_SESSIONS_START';
...
 export const getAllSessions = () =>
-    (dispatch) => {
+    (dispatch, getState) => {
         dispatch(createAction(UPDATE_SESSIONS_START)());
-        return getSessionsFromServer()
-            .then(sessions => dispatch(createAction(UPDATE_SESSIONS_COMPLETE)(sessions)))
+        return SessionsApi.getAllSessions()
+            .then(sessions => dispatch(createAction(UPDATE_SESSIONS_COMPLETE)(immutable.fromJS(sessions))))
+            .then(() => dispatch(createAction(UPDATE_LAST_CONTACT_START)()))
+            .then(CommsApi.getLastContacted)
```
Comment (lclarksl) [inline app/actions/index.js]: This is getLastContact now

File: app/components/SessionList.jsx [hunk 2]
```diff
@@ -20,6 +20,7 @@ class SessionList extends Component {
                               rating={session.speakerRating}
                               adminName={session.adminName}
                               adminImageUri={session.adminImageUrl}
+                              speakerLastContacted={session.speakerLastContacted}
```
Comment (lclarksl) [inline app/components/SessionList.jsx]: Should be lastContact

**Comment by lclarksl** [comment]
:shipit: after minor changes

**Comment by jebbinSCL** [comment]
:shipit:

**Pull request merged**